src/index.js

```diff
@@ -20,13 +22,21 @@ export default {
             throw new Error('Unsupported browser!');
 
         const browserNameString = browserName.match(/([^@:]+)/);
+        const browserFormalName = browserNameString[1].trim().toLocaleLowerCase();
         let version = browserName.match(/@([^:]+)/);
         let platform = browserName.match(/:(.+)/);
 
         version = version ? version[1] : undefined; // eslint-disable-line no-undefined
         platform = platform ? platform[1] : undefined; // eslint-disable-line no-undefined
-        const browser = await new Builder().forBrowser(browserNameString[1], version, platform).usingServer(this.seleniumServer).build();
 
+        const builder = new Builder().forBrowser(browserFormalName, version, platform).usingServer(this.seleniumServer);
```

Comment:
Why did you remove the await here?

Reply:
Here we seperate builder from build in order to add more options latter.
And here is when browser actually opened:
await builder.build()